src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs

```diff
@@ -34,8 +33,11 @@ public class AutoReviewController : Controller
             _reviewManager = reviewManager;
         }
 
+        // setReleaseTag param is set as true when request is originated from release pipeline to tag matching revision as released
+        // regular CI pipeline will not send this flag in request
+        [TypeFilter(typeof(ApiKeyAuthorizeAsyncFilter))]
```

Comment:
yes. We don't need api key auth to get a review status so it can be called without access to keyvault. It's a read only operation. Only upload and create modifies the review and needs auth.